conjur/util/init_utils.py

```diff
@@ -21,9 +24,9 @@ def validate_init_action_ssl_verification_input(ca_path, is_self_signed, ssl_ver
         use_ca_bundle = True
     options = [use_ca_bundle, is_self_signed, not ssl_verify]
```

Comment:
Consider using mutually exclusive in arg parse instead of this.
https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
Done